src/amo/sagas/reviews.js

```diff
@@ -60,6 +69,41 @@ function* fetchReviews({
   }
 }
 
+function* fetchRatingSummary({
+  payload: { errorHandlerId, addonId },
+}: FetchRatingSummaryAction): Generator<any, any, any> {
+  const errorHandler = createErrorHandler(errorHandlerId);
+  try {
+    const state = yield select(getState);
+
+    const params: GetReviewsParams = {
+      addon: addonId,
+      apiState: state.api,
+      show_grouped_ratings: true,
```

Comment:
I see in the api, it has a version number. Will we ever need that here? 

also, as far as I could tell, this is only in the api v4. So there is that dependency on upgrading in order to use this?

Reply:
This API endpoint is a bit overloaded. We don't need to filter by version to get the grouped ratings.

I tested the v3 endpoint and it works. We should probably figure out if we can upgrade to v4 soon. We probably can.